src/sync/mod.rs

```diff
@@ -1 +1,4 @@
+//! Synchronization primitives.
+
 pub mod list;
```

Comment:
The `crossbeam_epoch` crate implements epoch-based memory reclamation, but in doing so, it uses concurrent list and queue.  Thus it should implement them.  Then the `crossbeam` crate will depend on `crossbeam_epoch` and implement various data structures.  But list and queue are already implemented in `crossbeam_epoch`, so I thought `crossbeam` can reexport those implemented in `crossbeam_epoch`.  This is the reason I made `list` (and later `ms_queue`) public.  What do you think of this crate structure?